README.md

```diff
@@ -1,11 +1,8 @@
 <div align="center">
-  <br />
-  <img src="readme-header.svg" width="80%" alt="JoinMarket Web UI" />
-  <br />
+  <img src="docs/assets/readme-header-dark.svg#gh-light-mode-only">
+  <img src="docs/assets/readme-header-light.svg#gh-dark-mode-only">
 </div>
 
-<h1 align="center" style="font-weight: bold !important">JoinMarket Web UI</h1>
```

Comment:
Should we keep a title heading? Thinking of screen readers and search engines.
It would not look too bad visually before the description "Jam is a web UI for [...]". 
Just a thought. No blocker.

Reply:
Oh yeah, screen readers and search engines will pick up the `alt` texts (which I forgot to add back, but now they're there. 😄)